_data/schedule.yml

```diff
@@ -1,164 +1,154 @@
+  date: "Mar 6"
   materials: ~
-  title: "[Best Practices in Unity & VR](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Unity_Practices.pdf)"
+  title: "[Visual Perception](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Unity_Practices.pdf)"
 
 - 
-  date: "Oct 17"
+  date: "Mar 11"
   materials: "LaValle, Ch. 6"
-  title: "[Simulation in VR; Towards Realism](https://piazza.com/class/jl41yg3wwgr56m?cid=271)"
+  title: "VR and Unity Best Practices"
 
 - 
-  date: "Oct 22"
+  date: "Mar 13"
   materials: "LaValle, Ch. 6"
-  title: "[Color](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture12.pptx)"
+  title: "**No Lecture: Exam Week**"
 
 - 
-  date: "Oct 24"
-  materials: "LaValle, Ch. 7"
-  title: "[Visual Rendering](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture14.pptx)"
+  date: "Mar 18"
+  materials: ~
+  title: "**No Lecture: Spring Break**"
 
 - 
-  date: "Oct 29"
-  materials: "LaValle, Ch. 7"
-  title: "[Visual Rendering](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture14.pptx)"
+  date: "Mar 20"
+  materials: ~
+  title: "**No Lecture: Spring Break**"
 
 - 
-  date: "Oct 31"
+  date: "Mar 25"
   materials: "LaValle, Ch. 7"
-  title: "[Visual Rendering](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture14.pptx)"
+  title: "[Visual Rendering for VR](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture14.pptx)"
 
 - 
-  date: "Nov 5"
+  date: "Mar 27"
   materials: "LaValle, Ch. 7"
-  title: "[Visual Rendering for Virtual Reality](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture18.pptx)"
+  title: "[Motion in Real and Virtual Worlds](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture18.pptx)"
 
 - 
-  date: "Nov 7"
+  date: "Apr 1"
   materials: "LaValle, Ch. 8"
-  title: "[Motion in Real and Virtual Worlds](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture19.pptx)"
+  title: "[Tracking](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture19.pptx)"
 
 - 
-  date: "Nov 12"
+  date: "Apr 3"
   materials: "LaValle, Ch. 9, 12"
-  title: "[Motion in Real and Virtual Worlds](https://courses.engr.illinois.edu/cs498vr3/fa2018//lectures/Lecture21.pptx)"
+  title: "[Interaction](https://courses.engr.illinois.edu/cs498vr3/fa2018//lectures/Lecture21.pptx)"
 
 - 
-  date: "Nov 14"
+  date: "Apr 8"
   materials: "LaValle, Ch. 9"
-  title: "Tracking"
+  title: "Audio in VR"
 
 - 
-  date: "Nov 19"
+  date: "Apr 10"
   materials: ~
-  title: "No lecture - Thanksgiving Break"
+  title: "Survey of Current State of VR in Industry (Specifics regarding the Game Industry)"
 
 - 
-  date: "Nov 21"
+  date: "Apr 15"
   materials: ~
-  title: "No lecture - Thanksgiving Break"
+  title: "Design in more Detail (Mid-level processes)"
 
 - 
-  date: "Nov 26"
+  date: "Apr 17"
   materials: ~
-  title: ~
+  title: "Production in more Detail (Studios, Teams and lower level processes)"
```

Comment:
I advise against parenthetical clauses. Especially for the first one, a possible shortened version is "Survey of VR in the Game Industry". Also for all of these, the title case should be consistently applied (or not).

Reply:
Good suggestion - thanks